DotNetCasClient/Validation/CasSaml11Response.cs

```diff
@@ -98,7 +98,7 @@ private void ProcessValidAssertion()
         {
             protoLogger.Debug("Unmarshalling SAML response");
             XmlDocument document = new XmlDocument();
-            document.Load(new StringReader(_CasResponse));
+            document.LoadXml(_CasResponse);

```

Comment:
I don't have local CAS server to test with various settings, but it works with some 3rd party installation in the wild.

I don't expect any BOM issues for two reasons:

1. Method which call this one receives `string` by reading HTTP response without specifying Encoding; in such cases .NET treats request as UTF8 and omit BOM from resulted string

2. `LoadXml` basically does the same but in a safe way - it uses `StringReader` but disposes it. You can check in reference sources https://github.com/Microsoft/referencesource/blob/master/System.Xml/System/Xml/Dom/XmlDocument.cs#L1120

```
// Loads the XML document from the specified string.
public virtual void LoadXml( string xml ) {
    XmlTextReader reader = SetupReader( new XmlTextReader( new StringReader( xml ), NameTable ));
    try {
        Load( reader );
    }
    finally {
        reader.Close();
    }
}
```